src/generators/bendable-amogus/AmongUsBendableGenerator.res

```diff
@@ -0,0 +1,254 @@
+let requireImage = fileName => Generator.requireImage("./images/" ++ fileName)
+let requireTexture = fileName => Generator.requireImage("./textures/" ++ fileName)
+
+let id = "bendableamogusgenerator"
+
+let name = "Bendable Amogus generator"
+let thumbnail: Generator.thumnbnailDef = {
+  url: Generator.requireImage("./thumbnails/thumbnail.jpeg"),
```

Comment:
For consistency with other generators let's rename `thumbnails` to `thumbnail`

Reply:
Done